### PR TITLE
Fix installation issues for #15090

### PR DESCRIPTION
**Pull request by xu-yuan1**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *https://github.com/openvinotoolkit/openvino/issues/15090*

### CODE REVIEW

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 1]
```diff
@@ -96,22 +96,23 @@ To run the sample, you can use [public](@ref omz_models_group_public) or [Intel'
...
 ```sh
 build_samples.sh
 ```
 
 Once the build is completed, you can find sample binaries in the following folders:
-* C samples: `~/openvino_c_samples_build/intel64/Release`
-* C++ samples: `~/openvino_cpp_samples_build/intel64/Release`
+* C samples: `~/openvino_c_samples_build/<hardware_name>/Release`
+* C++ samples: `~/openvino_cpp_samples_build/<hardware_name>/Release`
+where the `hardware_name` is the output of `uname -m`, for example, `intel64`, `armhf`, or `aarch64`.
```
Comment (kblaszczak-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:115]:
```suggestion
* C samples: `~/openvino_c_samples_build/<architecture>/Release`
* C++ samples: `~/openvino_cpp_samples_build/<architecture>/Release`
where the <architecture> is the output of `uname -m`, for example, `intel64`, `armhf`, or `aarch64`.
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 2]
```diff
@@ -142,8 +142,8 @@ cd build
 make
 ```
 
-For the release configuration, the sample application binaries are in `<path_to_build_directory>/intel64/Release/`;
-for the debug configuration — in `<path_to_build_directory>/intel64/Debug/`.
+For the release configuration, the sample application binaries are in `<path_to_build_directory>/<hardware_name>/Release/`;
+for the debug configuration — in `<path_to_build_directory>/<hardware_name>/Debug/`.
```
Comment (ilya-lavrenov) [inline docs/OV_Runtime_UG/Samples_Overview.md:146]: => architecture 

in other places as well

Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:146]:
```suggestion
for the debug configuration — in `<path_to_build_directory>/<architecture>/Debug/`.
```

Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:146]:
```suggestion
for the debug configuration — in `<path_to_build_directory>/<architecture>/Debug/`.
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 3]
```diff
@@ -169,30 +169,31 @@ You can also build a generated solution manually. For example, if you want to bu
...
 
 To build the C or C++ sample applications for macOS, go to the `<INSTALL_DIR>/samples/c` or `<INSTALL_DIR>/samples/cpp` directory, respectively, and run the `build_samples.sh` script:
 ```sh
 build_samples.sh
 ```
 
 Once the build is completed, you can find sample binaries in the following folders:
-* C samples: `~/openvino_c_samples_build/intel64/Release`
-* C++ samples: `~/openvino_cpp_samples_build/intel64/Release`
+* C samples: `~/openvino_c_samples_build/<hardware_name>/Release`
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:188]:
```suggestion
* C samples: `~/openvino_c_samples_build/<architecture>/Release`
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 4]
```diff
@@ -169,30 +169,31 @@ You can also build a generated solution manually. For example, if you want to bu
...
+* C++ samples: `~/openvino_cpp_samples_build/<hardware_name>/Release`
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:189]:
```suggestion
* C++ samples: `~/openvino_cpp_samples_build/<architecture>/Release`
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 5]
```diff
@@ -169,30 +169,31 @@ You can also build a generated solution manually. For example, if you want to bu
...
+where the `hardware_name` is the output of `uname -m`, for example, `intel64` or `arm64`.
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:190]:
```suggestion
where the `<architecture>` is the output of `uname -m`, for example, `intel64` or `arm64`.
```

File: docs/OV_Runtime_UG/Samples_Overview.md (hunk 2 above)
Comment (kblaszczak-intel) [reply docs/OV_Runtime_UG/Samples_Overview.md:146]: we're on it ;)

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 6]
```diff
@@ -223,22 +223,23 @@ cd build
 make
 ```
 
-For the release configuration, the sample application binaries are in `<path_to_build_directory>/intel64/Release/`;
-for the debug configuration — in `<path_to_build_directory>/intel64/Debug/`.
+For the release configuration, the sample application binaries are in `<path_to_build_directory>/<hardware_name>/Release/`;
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:226]:
```suggestion
For the release configuration, the sample application binaries are in `<path_to_build_directory>/<architecture>/Release/`;
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 7]
```diff
@@ -223,22 +223,23 @@ cd build
...
+for the debug configuration — in `<path_to_build_directory>/<hardware_name>/Debug/`.
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:227]:
```suggestion
for the debug configuration — in `<path_to_build_directory>/<architecture>/Debug/`.
```

File: docs/OV_Runtime_UG/Samples_Overview.md [hunk 8]
```diff
@@ -142,8 +142,8 @@ cd build
 make
 ```
 
-For the release configuration, the sample application binaries are in `<path_to_build_directory>/intel64/Release/`;
-for the debug configuration — in `<path_to_build_directory>/intel64/Debug/`.
+For the release configuration, the sample application binaries are in `<path_to_build_directory>/<hardware_name>/Release/`;
```
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:145]:
```suggestion
For the release configuration, the sample application binaries are in `<path_to_build_directory>/<architecture>/Release/`;
```

File: docs/OV_Runtime_UG/Samples_Overview.md (hunk 2 above)
Comment (sgolebiewski-intel) [inline docs/OV_Runtime_UG/Samples_Overview.md:146]:
```suggestion
for the debug configuration — in `<path_to_build_directory>/<architecture>/Debug/`.
```

**Review by kblaszczak-intel — APPROVED** [review]

**Pull request merged**